packages/gatsby-plugin-image/src/babel-helpers.ts

```diff
@@ -5,6 +5,9 @@ import { getAttributeValues } from "babel-jsx-utils"
 
 export const SHARP_ATTRIBUTES = new Set([
   `src`,
+  `layout`,
```

Comment:
Is there a point to moving the existing strings around? Maybe try not to cause them to appear as deleted lines? :shrug:

Reply:
I reordered them to match the order in the resolver, which makes it easier to keep track of them